setup.py

```diff
@@ -31,14 +36,15 @@
             'backports.tempfile==1.0rc1',
             'cloudpickle>=0.1.0',
             'futures>=3.0.1',
-            'pylint',
+            'pylint~=2.7',
             'pylzma',
             'memory-profiler>=0.47',
             'pycodestyle',
             'pytest',
             'pytest-cov',
             'isort',
             'tornado>=4.3',
+            'parameterized>=0.7.4',
```

Comment:
I have the same question as @svaningelgem . Is it possible to avoid this extra dependency using pytest mechanisms?

Reply:
I looked around and did not find something that would do what I was trying to achieve. It is very useful as there are a lot of cases to test for many SQL functions so I really appreciate the added value of the lib